swift/apple/README.md

```diff
@@ -2,28 +2,36 @@
 
 Firezone clients for macOS and iOS.
 
-## Pre-requisites
+This document is intended as a reference for developers working on the Apple
+clients.
 
+## Prerequisites
+
+1. Ensure you have the latest stable version of Xcode installed and selected.
 1. Rust: `curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh`
 1. Request your Firezone email added to our Apple Developer Account
 1. Open Xcode, go to Settings -> Account and log in.
 
 If you're working on the macOS client, you'll need to disable SIP and enable
 system extension development mode:
 
-1. Follow [these instructions](https://developer.apple.com/documentation/security/disabling-and-enabling-system-integrity-protection) to disable SIP.
+1. Follow
+   [these instructions](https://developer.apple.com/documentation/security/disabling-and-enabling-system-integrity-protection)
+   to disable SIP.
 1. After that's complete, turn on system extension development mode:
 
 ```bash
 systemextensionsctl developer on
 ```
 
-This will prevent macOS from blocking the Network Extension from loading due to notarization or filepath restrictions.
+This will prevent macOS from blocking the Network Extension from loading due to
+notarization or filepath restrictions.
 
 **Be sure to re-enable SIP to test the app in a production-like environment.**
 
-You may consider using a macOS VM (such as Parallels Desktop) to develop the macOS client, as it's easier to
-disable SIP, take snapshots, and muck around with your system configuration without risking your main machine.
+You may consider using a macOS VM (such as Parallels Desktop) to test the
+standalone macOS client, as it can be easier to test different macOS versions
+and configurations without risking your main machine.
```

Comment:
I just tried to develop in a macOS VM and it's not possible because:

- Xcode signing into Apple ID doesn't work
- Xcode needs to register a hardware device for development